apps/easypid/src/features/activity/activityRecord.ts

```diff
@@ -1,17 +1,18 @@
 import { utils } from '@credo-ts/core'
-import type { AppAgent } from '@easypid/agent'
 import {
   type CredentialForDisplayId,
   type CredentialsForProofRequest,
-  type DisplayImage,
-  type FormattedSubmission,
   type FormattedTransactionData,
-  getDisclosedAttributeNamesForDisplay,
-  getUnsatisfiedAttributePathsForDisplay,
   getWalletJsonStore,
   useWalletJsonRecord,
 } from '@package/agent'
-
+import type { BaseAgent } from '@paradym/wallet-sdk/src/agent'
+import {
+  getDisclosedAttributeNamesForDisplay,
+  getUnsatisfiedAttributePathsForDisplay,
+} from '@paradym/wallet-sdk/src/display/common'
+import type { DisplayImage } from '@paradym/wallet-sdk/src/display/credential'
+import type { FormattedSubmission } from '@paradym/wallet-sdk/src/format/submission'
```

Comment:
What is the idea for users of the SDK? Will they import just from `@paradym/wallet-sdk` and are these internal utils we use?

Reply:
Same as the comment above.

Types might always be exported from `src/index.ts`, but I need to find the best pattern for that.